cluster/gce/gci/configure-helper.sh

```diff
@@ -1606,11 +1606,10 @@ EOF
 #
 # $1 is the file to create.
 # $2: the log owner uid to set for the log file.
-# $3: the log owner gid to set for the log file.
 function prepare-log-file {
   touch "$1"
-  chmod 644 "$1"
-  chown "${2:-${LOG_OWNER_USER:-root}}":"${3:-${LOG_OWNER_GROUP:-root}}" "$1"
+  chmod 640 "$1"
+  chown "${2:-root}":"${KUBE_POD_LOG_READERS_GROUP:-root}" "$1"
```

Comment:
Is `KUBE_POD_LOG_READERS_GROUP` set on GCE? What does this change imply to GCE users? Shall we add a if condition and keep the old behavior if `KUBE_POD_LOG_READERS_GROUP` is not set?